### DISCUSSION TITLE
Add FAQ document to Academy docs

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#7] [Bugs and Reports]
Please add FAQ document to be downloadable in academy docs

[BetfinPlatfromFAQ v1.0.pdf](https://github.com/user-attachments/files/17111299/BetfinPlatfromFAQ.v1.0.pdf)

**Comment by bf-monster** [comment]
<img width="710" alt="Screenshot 2024-09-24 at 14 45 46" src="https://github.com/user-attachments/assets/77bd1895-a5a6-42c2-9747-6c740664be46">
added

**Reply by bf-monster** [reply]
Close the issue, if you are satisfied, pls
@bf-mike-mkt

**Reply by bf-mike-mkt** [reply]
pls call it "Betfin Platform FAQ" and delete those bullet points. Other than that good
